src/improvements/tasks/MultisigTask.sol

```diff
@@ -237,43 +230,35 @@ abstract contract MultisigTask is Test, Script, StateOverrideManager, TaskManage
     }
 
     /// @notice Get the hash for this safe transaction.
-    function getHash(bytes memory callData, address safe) public view returns (bytes32) {
-        return keccak256(getEncodedTransactionData(safe, callData));
-    }
-
-    /// @notice Helper function to generate the approveHash calldata to be executed by child multisig owner on parent multisig.
-    function generateApproveMulticallData(Action[] memory actions) public view returns (bytes memory) {
-        bytes memory callData = getMulticall3Calldata(actions);
-        bytes32 hash = getHash(callData, parentMultisig);
-        IMulticall3.Call3Value memory call = IMulticall3.Call3Value({
-            target: parentMultisig,
-            allowFailure: false,
-            value: 0,
-            callData: abi.encodeCall(IGnosisSafe(parentMultisig).approveHash, (hash))
-        });
-
-        IMulticall3.Call3Value[] memory calls = new IMulticall3.Call3Value[](1);
-        calls[0] = call;
-        return abi.encodeWithSignature("aggregate3Value((address,bool,uint256,bytes)[])", calls);
+    function getHash(
+        bytes memory callData,
+        address safe,
+        uint256 value,
+        uint256 originalNonce,
```

Comment:
Why is this called originalNonce, and not just nonce?

Reply:
I reckon it is because before the nonce to encode with was a state variable, so it needed to be stored with more context.

Reply:
Same as the other comment around bases naming convention.